### PR TITLE
Fix the tag in openAPI

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Some endpoints were not being tagged with the right tag. This PR fixes it

## Recommended reading order
1. `openapi.yml`
2. The other files are propagated the change

### CODE REVIEW

**Review by colesnodgrass — APPROVED** [review]
Code changes seem fine.  I don't know enough about the context of this change to understand exactly what this is fixing.

File: airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+public class DbMigrationApiController {
```
Comment (alovew) [inline airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java:7]: what is this for?

Comment (benmoriceau) [reply airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java:7]: that's a mistake, It was in my untracked git change and sneak in the PR. I'll remove it once I have a green build.

**Comment by benmoriceau** [comment]
> enough

@colesnodgrass the context is explained in https://airbytehq-team.slack.com/archives/C03AS1GAQV6/p1666722517579499 it happened in another PR

File: airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java (hunk 1 above)
Comment (alovew) [reply airbyte-server/src/main/java/io/airbyte/server/apis/DbMigrationApiController.java:7]: 👍 this looks good to me otherwise so i'll approve

**Review by alovew — APPROVED** [review]

**Comment by benmoriceau** [comment]
This will break the cloud project, I'll prepare a follow up PR.

**Pull request merged**